presto-tests/src/test/java/io/prestosql/tests/TestInformationSchemaConnector.java

```diff
@@ -192,6 +192,12 @@ public void testMetadataCalls()
                         .withListSchemasCount(1)
                         .withListTablesCount(2)
                         .withGetColumnsCount(10000));
+        assertNoMetadataCalls("SELECT count(*) from test_catalog.information_schema.tables WHERE table_schema = ''", "VALUES 0");
```

Comment:
I was also thinking about:
```
        assertFalse(searchFrom(getQueryRunner().createPlan(getSession(), "SELECT count(*) from test_catalog.information_schema.tables WHERE table_schema = ''", WarningCollector.NOOP).getRoot())
                .where(TableScanNode.class::isInstance)
                .findFirst()
                .isPresent());
```

Reply:
Thanks for your insights. I have applied your comments